### PR TITLE
Rapidpro integration code refactor

### PR DESCRIPTION
**Pull request by ellykits**

### CODE REVIEW

File: src/main/java/org/opensrp/web/Constants.java [hunk 1]
```diff
@@ -19,16 +21,16 @@
...
-    public static final String PAGE_SIZE = "pageSize";
+    String PAGE_SIZE = "pageSize";
 
-    public static final String ORDER_BY_TYPE = "orderByType";
+    String ORDER_BY_TYPE = "orderByType";
 
-    public static final String ORDER_BY_FIELD_NAME = "orderByFieldName";
+    String ORDER_BY_FIELD_NAME = "orderByFieldName";
 
     String LOCATIONS = "locations";
```
Comment (ndegwamartin) [inline src/main/java/org/opensrp/web/Constants.java:35]: Any reason we are changing the access modifies for these fields?

Comment (ellykits) [reply src/main/java/org/opensrp/web/Constants.java:35]: I performed code clean up on the code. Those are access modifiers applied in a public interface which is not necessary.

File: pom.xml [hunk 2]
```diff
@@ -26,8 +26,8 @@
         <redis.lettuce.version>5.2.2.RELEASE</redis.lettuce.version>
         <opensrp.updatePolicy>always</opensrp.updatePolicy>
         <nexus-staging-maven-plugin.version>1.5.1</nexus-staging-maven-plugin.version>
-        <opensrp.core.version>2.12.2-SNAPSHOT</opensrp.core.version>
```
Comment (Rkareko) [inline pom.xml:29]: Is there a reason why the version is jumping from `2.12.2` to `2.12.4` ?

File: src/main/java/org/opensrp/web/Constants.java [hunk 3]
```diff
@@ -70,4 +72,11 @@
         }
     }
 
+    interface RapidPro {
+	    String CONTACT = "contact";
```
Comment (Rkareko) [inline src/main/java/org/opensrp/web/Constants.java:76]: Nitpick ..... check on identation

**Comment by ellykits** [comment]
Closing this. Will create a PR from a clone branch of `rapidpro-refactor`

**Pull request closed without merging**